src/Microsoft.AspNet.Mvc.ModelBinding/ModelMetadataTypeAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Microsoft.AspNet.Mvc
```

Comment:
did we have a discussion about the appropriate name and namespace for this attribute?  could for example have provided a .NET Core-only `MetadataTypeAttribute` in the `System.ComponentModel.DataAnnotations` namespace and used the system-provided attribute when available.